modules/picard/addorreplacereadgroups/main.nf

```diff
@@ -43,7 +43,7 @@ process PICARD_ADDORREPLACEREADGROUPS {
         -PL ${PLATFORM} \\
         -PU ${BARCODE} \\
         -SM ${SAMPLE} \\
-        -CREATE_INDEX true
+        --CREATE_INDEX true
```

Comment:
Should we have that in args?
I assume there might be some cases when we actually don't want index

Reply:
I have no opinion, I'm just updating everything to the latest notation.

Reply:
I'll create an issue for it then, and we'll figure it out in a separate PR

Reply:
cf #1588